### ISSUE TITLE
Add a definition of "accessibility" to the write up

### ISSUE THREAD

**Issue by jwestw** [ONSdigital/SDG_11.2.1#394]
In the [definitions](https://onsdigital.github.io/SDG_11.2.1/html/write_up.html#definitions) section of the write up, there are a few key terms that should be defined. One of these is "accessibility". If this is defined elswhere in the write up then maybe it should be moved here and referenced.

**Comment by paigeh1** [comment]
@jwestw not sure where the definitions are in the write up?
